src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -410,25 +414,67 @@ public MonetaryAmount summarizeSubTotals(List<InvoiceLine> lines, CurrencyUnit c
       .collect(toList());
   }
 
+  private void sortByInvoiceLineNumber(List<InvoiceLine> lines) {
+    lines.sort(Comparator.comparing(invoiceLine -> Integer.parseInt(invoiceLine.getInvoiceLineNumber().split(DASH_SEPARATOR)[1])));
```

Comment:
Better to link this and `buildInvoiceLineNumber` methods to highlight dependency...